homeassistant/components/sensehat/sensor.py

```diff
@@ -43,9 +43,8 @@
 
 def get_cpu_temp():
     """Get CPU temperature."""
-    res = os.popen("vcgencmd measure_temp").readline()
-    t_cpu = float(res.replace("temp=", "").replace("'C\n", ""))
-    return t_cpu
+    t_cpu = os.popen("cat /sys/class/thermal/thermal_zone0/temp").readline()
```

Comment:
why not just read it?

Reply:
I wanted to keep the same structure of before. But please propose a modification if you know a better way, thanks!

Reply:
i.e. Path("/sys/class/thermal/thermal_zone0/temp").read_text() - now you create a subprocess (copy the hole memory of the current process) which is darm slow and take a lot of resource for just read a value from a file.

Reply:
testing right now. Thanks!